src/dotnet/commands/dotnet-list/dotnet-list-p2ps/LocalizableStrings.cs

```diff
@@ -0,0 +1,7 @@
+namespace Microsoft.DotNet.Tools.List.ProjectToProjectReferences
+{
+    internal class LocalizableStrings
+    {
+        public const string NoReferencesFound = "There are no {0} references in project {1}. ;; {0} is the type of the item being requested (project, package, p2p) and {1} is the object operated on (a project file or a solution file). ";
```

Comment:
Why the double ;; in the middle of the string? Also, why does this string terminate in an empty space?